ts/session/apis/open_group_api/opengroupV2/ApiUtil.ts

```diff
@@ -39,8 +39,8 @@ export type OpenGroupV2InfoJoinable = OpenGroupV2Info & {
 // tslint:disable: no-http-string
 
 const legacyDefaultServerIP = '116.203.70.33';
-const defaultServer = 'https://open.getsession.org';
-const defaultServerHost = new window.URL(defaultServer).host;
+export const defaultServerSession = 'https://open.getsession.org';
```

Comment:
defaultSessionServer?

Reply:
this is the url  of the server where `or join one of these` rooms comes from. Do you have a better name? :)

Reply:
Yes, it's defaultSessionServer 😂  Sorry the question caused confusion, I was suggesting this name. 🙏